projects/admin/src/app/record/detail-view/item-type-detail-view/item-type-detail-view.component.html

```diff
@@ -0,0 +1,67 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019 RERO
+  
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+  
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+  
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record$ | async as record">
+  <h1 class="mb-5">{{ record.metadata.name }}</h1>
+  <dl class="row">
+    <!-- DESCRIPTIION -->
+    <ng-container *ngIf="record.metadata.description">
+      <dt class="col-3 label-title" translate>Description</dt>
```

Comment:
why not `font-weight-bold` as previous?

Reply:
because "bold" is the default behavior for `dt` tags